digits/model/images/job.py

```diff
@@ -17,3 +21,65 @@ def __init__(self, **kwargs):
         super(ImageModelJob, self).__init__(**kwargs)
         self.pickver_job_model_image = PICKLE_VERSION
 
+    def get_snapshot_filename(self,epoch=-1):
```

Comment:
since you're creating this method you might as well move it to `digits/model/tasks/train.py`, replace the code from [there](https://github.com/NVIDIA/DIGITS/blob/v4.0.0-rc.3/digits/model/tasks/torch_train.py#L929-L946) and [there](https://github.com/NVIDIA/DIGITS/blob/v4.0.0-rc.3/digits/model/tasks/caffe_train.py#L1321-L1330) with a call to this new method, and perhaps add unit tests too.